src/main/webapp/js/additionalQuestionInfo.js

```diff
@@ -1,16 +1,12 @@
-/*
-This function is called to toggle the visibility of additional question information for the specified question.
-*/
-function toggleAdditionalQuestionInfo(identifier){
-    console.log($("#questionAdditionalInfoButton-"+identifier).text());
-    console.log("#questionAdditionalInfoButton-"+identifier);
+// This function is called to toggle the visibility of additional question information for the specified question.
```

Comment:
'This function is called to' part is redundant. Can be 'Toggles the visibility ...'